ci/other/pylintrc

```diff
@@ -113,7 +113,10 @@ disable=
     too-many-instance-attributes,
     too-many-locals,
     too-many-public-methods,
-    too-many-statements
+    too-many-statements,
+    too-many-lines,
+    duplicate-code,
+    cyclic-import
```

Comment:
Is cyclic import not a valid thing to consider?

Reply:
Good point, however there are over 90 cyclic-import errors reported.  Analyzing the python software architecture and possibly refactoring it to eliminate cyclic imports is beyond the scope of this PR.  I'll file an issue to see if one of the python major code owners will look into this.

Reply:
Sounds good, thanks